cmd/cli/version/version.go

```diff
@@ -105,12 +109,67 @@ func (oV *VersionOptions) Run(ctx context.Context, cmd *cobra.Command) error {
 		serverVersion, err := util.GetAPIClient(ctx).Version(ctx)
 		if err != nil {
 			log.Ctx(ctx).Error().Err(err).Msgf("could not get server version")
-			return err
+			// Fail silently
+		} else {
+			versions.ServerVersion = serverVersion
+			columns = append(columns, serverVersionColumn)
 		}
+	}
```

Comment:
You've put this back to what it was before. It needs to return the error if you can't get the requester (server) version, but not the update version.